packages/vaadin-icon/src/vaadin-icon.js

```diff
@@ -83,7 +124,7 @@ class IconElement extends ThemableMixin(ElementMixin(PolymerElement)) {
        */
```

Comment:
Naming this property `size` was confusing for me. At first I though this is the pixel size (width and height) of the icon on the screen. Now I realize it's actually the SVG viewBox width and height and as such it has nothing to do with the size of the icon on the screen.

Is this just me? What could be some other way to call this property?

Reply:
I don't have any better ideas, but I will add this to JSDoc for the `size` property.